src/components/Post.js

```diff
@@ -5,36 +5,35 @@ import { useNavigate } from "react-router-dom";
 // import axios from "axios";
 
 const Post = () => {
-  const [blog, setBlog] = useState({ title: "", content: "", author: "" });
-  const [postErr, setPostErr] = useState("")
+  const [blog, setBlog] = useState({ title: "", content: "" });
+  const [postErr, setPostErr] = useState("");
   // const [isAuth, setIsAuth] = useState("") //state to check if user have access to post
 
-
-  
   const handleBlogDetails = (e) => {
     e.preventDefault();
     setBlog((prev) => ({ ...prev, [e.target.name]: e.target.value }));
     // setIsAuth("You are not allowed to post until you are logged in")
   };
 
+  
+
   let navigate = useNavigate();
+
   const handleSubmit = async (e) => {
     e.preventDefault();
+    const token = JSON.parse(localStorage.getItem("user-data")); // to get the token saved in the local storage
```

Comment:
Should this not be called `userData` instead of `token`?

Reply:
I will change this. I just felt since there are access token and refresh token inside the user-data, I just decided to give it token.